pipelines/consolidate_injected_deepCoadd_catalogs.yaml

```diff
@@ -0,0 +1,4 @@
+description: Consolidation of per-band and per-patch catalogs into tract-level catalogs.
+tasks:
+  consolidate_injected_catalogs:
```

Comment:
This task hasn't been explicitly named anywhere in the docs AFAICT. Please make a reference.

This will be more important if this task is also moved into a common YAML (see next comment).